### PR TITLE
[New Tool] Integron Finder (Pasteur)

### PR DESCRIPTION
**Pull request by Raynooo**
# Submitting a Container
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [X] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [X] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [X] LABEL base_image  
 - [X] LABEL version
 - [X] LABEL software.version  
 - [X] LABEL about.summary  
 - [X] LABEL about.home  
 - [X] LABEL about.license  
 - [X] LABEL extra.identifier  
 - [X] MAINTAINER  
 
 3. Extra
  - [ ] I have written tests in test-cmds.txt
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata

### CODE REVIEW

**Comment by osallou** [comment]
please specify in about.license_file the location of the license file in the container, or a url to license for this release of the software.

**Comment by osallou** [comment]
No biotools label defined, please check if tool is not already defined in biotools (https://bio.tools) and add extra.identifier.biotools label if it exists. If it is not defined, you can ignore this comment.

**Comment by osallou** [comment]
please specify in about.license_file the location of the license file in the container, or a url to license for this release of the software.

**Comment by osallou** [comment]
No biotools label defined, please check if tool is not already defined in biotools (https://bio.tools) and add extra.identifier.biotools label if it exists. If it is not defined, you can ignore this comment.

**Comment by osallou** [comment]
No integron-finder/1.5.1/test-cmds.txt (test file) present, skipping tests

**Comment by osallou** [comment]
please specify in about.license_file the location of the license file in the container, or a url to license for this release of the software.

**Comment by osallou** [comment]
No integron-finder/1.5.1/test-cmds.txt (test file) present, skipping tests

**Comment by osallou** [comment]
please specify in about.license_file the location of the license file in the container, or a url to license for this release of the software.

**Comment by osallou** [comment]
No integron-finder/1.5.1/test-cmds.txt (test file) present, skipping tests

File: integron-finder/1.5.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	software="Integron Finder" \ 
+    version="1" \
+	software.version="1.5.1" \ 
+	about.summary="Integron Finder aims at detecting integrons in DNA sequences." \ 
+	base_image="ubuntu:16.04" \
+    about.home="https://github.com/gem-pasteur/Integron_Finder" \
+    about.license="SPDX:GPL-3.0-only" \
+    extra.identifier.biotools="integron_finder" \ 
+    ANNOT.Name="Integron_Finder" \
+    ANNOT.Version="1.5.1" \
```
Comment (osallou) [inline integron-finder/1.5.1/Dockerfile]: could do some cleanup to avoid duplication in labels

Comment (Raynooo) [reply integron-finder/1.5.1/Dockerfile]: I was wondering if we wanted to keep those "ANNOT..." labels or if we don't care about them. If we don't care I'll remove them here (and in a couple other tools where they went unnoticed ;) )

Comment (osallou) [reply integron-finder/1.5.1/Dockerfile]: those ANNOT were bioshadocker related. EDMA ones could be kept, for record, but name, version, ... are duplicates with our LABEL

**Comment by osallou** [comment]
please specify in about.license_file the location of the license file in the container, or a url to license for this release of the software.

**Comment by osallou** [comment]
No integron-finder/1.5.1/test-cmds.txt (test file) present, skipping tests

**Comment by osallou** [comment]
please specify in about.license_file the location of the license file in the container, or a url to license for this release of the software.

**Comment by osallou** [comment]
No integron-finder/1.5.1/test-cmds.txt (test file) present, skipping tests

**Pull request merged**